src/type/schema.js

```diff
@@ -72,6 +72,62 @@ export function assertSchema(schema: mixed): GraphQLSchema {
  *       mutation: MyAppMutationRootType,
  *     })
  *
+ * Note: When the schema is constructed, only the types that are reachable through
+ * traversing the root types & those that are explicitly referenced are included.
+ * Example:
+ *      const {
+ *        GraphQLID,
+ *        GraphQLInterfaceType,
+ *        GraphQLNonNull,
+ *        GraphQLObjectType,
+ *        GraphQLSchema,
+ *        printSchema,
+ *      } = require('graphql')
```

Comment:
I think it should be shorter + most of our examples are based on SWAPI example.
So how about this?
```js
const characterInterface = new GraphQLInterfaceType({
  name: 'Character',
  ...
});

const humanType = new GraphQLObjectType({
  name: 'Human',
  interfaces: [characterInterface],
  ...
});

const droidType = new GraphQLObjectType({
  name: 'Droid',
  interfaces: [characterInterface],
  ...
});

const schema = new GraphQLSchema({
  query: new GraphQLObjectType({
    name: 'Query',
    fields: {
      hero: { type: characterInterface, ... },
    }
  }),
  ...
  // Since this schema reference only `Character` interface it's nessary to
  // explicitly list types that implement it.
  types: [humanType, droidType],
})
```

Reply:
@IvanGoncharov You're right. This would keep things more coherent.

Reply:
@IvanGoncharov Wouldn't the sentence in the schema example only be valid if the user explicitly wants those types (`humanType`, `driodType`) to be included in the final schema?

Reply:
> In which case I would like to rewrite it as such:

@buoyantair Excellent correction 👍